google/google_test.go

```diff
@@ -0,0 +1,56 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSetupFromString(t *testing.T) {
+	t.Run("should assign store and config accordingly", func(t *testing.T) {
+		store = nil
+		conf = nil
+		SetupFromString("http://fake.fake", "clientid", "clientsecret", []string{}, []byte("secret"))
+		assert.NotNil(t, conf)
+		assert.NotNil(t, store)
+		assert.Equal(t, conf.ClientID, "clientid")
+		assert.Equal(t, conf.ClientSecret, "clientsecret")
+	})
+}
+
+func TestWithLoginURL(t *testing.T) {
+
+	var testCases = []struct {
+		description    string
+		urlParm        string
+		expectUrlLogin string
```

Comment:
Linter says we should use upper case URL.
I know we don't follow it elsewhere but let's follow the convention for new things.

Reply:
Aye, will do!